### PR TITLE
Update logo/favicons

### PR DESCRIPTION
**Pull request by eliwilliamson**

### CODE REVIEW

**Comment by eliwilliamson** [comment]
@luise should be good to go, look it over and merge if all is 💯

**Comment by luise** [comment]
Thanks, @eliwilliamson ! Seeing it on the page makes me think that the boat and Kelda are too close to each other. I'll update the spacing and send you an email with the new files!

**Comment by eliwilliamson** [comment]
@luise spacing should be all updated 👌

File: site/layouts/partials/header.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/img/favicon/favicon-32x32.png">
```
Comment (luise) [inline site/layouts/partials/header.html]: Is there a reason you kept the old android-chrome-48x48.png, now that we have android-chrome-192x192.png? Should we delete the 48x48 file or update it to be the new logo?

**Comment by eliwilliamson** [comment]
I apologize - that snuck through. Yes we can delete that 👍

-Eli

Eli Williamson
eliwilliamson.com <http://eliwilliamson.com/>
(513) 504-0921

**Comment by luise** [comment]
@eliwilliamson Thanks, this looks great! I just left one comment about the android favicon. When that's resolved, we're ready to merge!

**Comment by luise** [comment]
Sorry, didn't see your reply! I'll go ahead and merge!

**Pull request merged**